include/ast/type.h

```diff
@@ -173,6 +174,19 @@ class TableType {
   /// @}
 };
 
+class Table {
```

Comment:
For this object which contains the expression and type, you should add it in the `Segment`, such as the `GlobalSegment` and `ElementSegment`.